packages/components/src/components/form/Switch/Switch.tsx

```diff
@@ -71,52 +80,79 @@ const CheckboxInput = styled.input`
     width: 1px;
 `;
 
+const Label = styled.div<Pick<SwitchProps, 'isDisabled' | 'isAlert' | 'isSmall'>>`
+    color: ${({ isAlert, isDisabled, theme }) =>
+        getLabelColor(theme, { alert: isAlert, disabled: isDisabled })};
+    text-align: left;
```

Comment:
I believe `left` is the default value of `text-align`

Reply:
9ada0fde38e5e788ad4b7eba036e50f07baef91f